src/components/core/base/VisualizationLoadingHOC.tsx

```diff
@@ -281,15 +288,42 @@ export function visualizationLoadingHOC<
             }
         };
 
+        private getAllMeasureHeaderItems(
+            response: Execution.IExecutionResponse,
+        ): Execution.IMeasureHeaderItem[] {
+            return flatten(
+                flatten(
+                    flatten(response.dimensions).map(
+                        (dimension: Execution.IResultDimension) => dimension.headers,
+                    ),
+                )
+                    .filter((header: Execution.IHeader) => Execution.isMeasureGroupHeader(header))
+                    .map((header: Execution.IMeasureGroupHeader) => header.measureGroupHeader.items),
+            );
+        }
+
+        private getPossibleDrillableItems(response: Execution.IExecutionResponse): IDrillableItemPushData[] {
+            return this.getAllMeasureHeaderItems(response).map(
+                (measure: Execution.IMeasureHeaderItem): IDrillableItemPushData => ({
+                    type: "measure",
+                    localIdentifier: measure.measureHeaderItem.localIdentifier,
+                    title: measure.measureHeaderItem.name,
```

Comment:
I think measures use both `name` and `title`. In AD/Kd `title` is used as custom name for the measure... Please investigate this. Maybe I'm wrong.